Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -188,8 +196,100 @@ function publishPackageUsingArtifactTool(
         return;
     }
 
+    //Retry if package exist error occurs
+    if (execResult.code == 17) {
+        return execResult.code;
+    }
+
     telemetry.logResult("Packaging", "UniversalPackagesCommand", execResult.code);
     throw new Error(tl.loc("Error_UnexpectedErrorArtifactTool",
         execResult.code,
         execResult.stderr ? execResult.stderr.trim() : execResult.stderr));
 }
+
+async function getLatestPackageVersion(
```

Comment:
shouldn't this be "get*Next*PackageVersion"? or am I misunderstanding?

Reply:
Yup you are right, it gets the current latest version and increment it, changed the name